tests/tests/property_backends.rs

```diff
@@ -22,17 +19,41 @@ pub enum Visibility {
     Private,
 }
 
+impl Property for Visibility {
+    fn into_value(&self) -> Result<Option<PropertyValue>, PropertyError> {
```

Comment:
Consider switching this to use strum Display/EnumString to save the boilerplate https://crates.io/crates/strum